includes/templates/admin/setup/membership-check.php

```diff
@@ -15,9 +15,12 @@
 			<?php
```

Comment:
Let's rename the file `site-check.php` since this incorporates more than membership now.

Change the option `gatherpress_suppress_membership_notification` to `gatherpress_suppress_site_notification`. We can update this change in [GatherPress Alpha](https://github.com/GatherPress/gatherpress-alpha).